### PR TITLE
fix(docs): fix bad speaker note

### PR DESCRIPTION
**Pull request by ThibDujardin**

### CODE REVIEW

File: docs/markdown/30-testing/01-bats-setup.md [hunk 1]
```diff
@@ -28,6 +28,4 @@ Notes:
 
 Easy way to use TDD method during development
 
-Notes:
-
 Speaker **Thibauld**
```
Comment (rlespinasse) [inline docs/markdown/30-testing/01-bats-setup.md:31]: put speaker line as 1st line of the notes block

File: docs/markdown/50-lifecycle/00-release.md [hunk 2]
```diff
@@ -9,6 +9,8 @@ Notes:
 
 Speaker **Romain**
 
+##--##
+
```
Comment (rlespinasse) [inline docs/markdown/50-lifecycle/00-release.md:13]: Remove this and the previous block `notes`, this is one slide and not two

File: docs/markdown/30-testing/01-bats-setup.md (hunk 1 above)
Comment (ThibDujardin) [reply docs/markdown/30-testing/01-bats-setup.md:29]: Done.

File: docs/markdown/50-lifecycle/00-release.md (hunk 2 above)
Comment (ThibDujardin) [reply docs/markdown/50-lifecycle/00-release.md]: Done.

**Review by rlespinasse — APPROVED** [review]

**Pull request merged**